patches/@sentry+utils+7.30.0.patch

```diff
@@ -0,0 +1,28 @@
+diff --git a/node_modules/@sentry/utils/cjs/misc.js b/node_modules/@sentry/utils/cjs/misc.js
```

Comment:
If we need such a hack, then we really need to leave some comments here with links to opened issue in sentry repo.